packages/babel-parser/test/fixtures/typescript/cast/destructure-and-assign/input.ts

```diff
@@ -0,0 +1 @@
+(a as number) = 42;
```

Comment:
Shouldn't this be:

```
[a as number] = 42
```

?

Reply:
It sure should be. Was playing around with input and forgot to change it back. Thanks for noticing!